cantabular-import/analysis/full-import-export/main.go

```diff
@@ -252,13 +252,13 @@ func main() {
 		if instanceFromAPI.Version.State == "edition-confirmed" {
 			// fmt.Printf("\ninstanceFromAPI: %v\n", instanceFromAPI)
 			// spew.Dump(instanceFromAPI)
-			fmt.Printf("Got 'edition-confirmed' after: %d milliseconds\n", 100*(51-attempts))
+			fmt.Printf("Got 'edition-confirmed' after: %d milliseconds\n", 100*(201-attempts))
```

Comment:
if we have a `MaxAttempts` const, here we could do `...(MaxAttempts+1-attempts)...`